Week-03/css4.html

```diff
@@ -0,0 +1,59 @@
+<html>
+<head>
+<style>
+
+/* default setup for div blocks */
+.block {
+    width: 40px;
+    height: 40px;
+    background-color: white;
+    border-width: 1px;
+    border-color: gray;
+    border-style: dashed;
+    display: inline-block;
+}
+/* a yellow block */
+.yellow {
+    background-color: yellow;
+    width: 40px;
+    height: 40px;
+    border-width: 1px;
+    border-color: gray;
+    border-style: dashed;
+    display: inline-block;
+    color: yellow;
+}
+/* a black block */
+.black {
+    background-color: black;
+    width: 40px;
+    height: 40px;
+    border-width: 1px;
+    border-color: gray;
+    border-style: dashed;
+    display: inline-block;
+}
+
+.blocks-container {
+    width: 210px;
+}
+
+</style>
+</head>
+<body>
+    <section class="blocks-container">
+        <div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div>
+        <div class="yellow" class="block">b</div><div class="black" class="block">b</div><div class="yellow" class="block">b</div><div class="black" class="block">b</div><div class="yellow" class="block">b</div>
+        <div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div>
+        <div class="yellow" class="block">b</div><div class="black" class="block">b</div><div class="yellow" class="block">b</div><div class="black" class="block">b</div><div class="yellow" class="block">b</div>
+        <div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="black" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div>
+    </section>
+
+    <section>
+        <p>Let's draw a smiley face.</p>
+        <p><b>The challenge:</b> Add a class to the necessary <code>div</code> elements that will turn the background color black. Add this to the elements necessary such that you draw two eyes and a smile.</p>
+        <p><b>Advanced challenge:</b> Make all of the other blocks have a yellow background. There is an efficient way to do this, and an ineffecient way. What is the efficient way?</p>
+        <p><b>Advanced challenge:</b> The letter <code>b</code> is still in the way and messing up our drawing. Can we use CSS to change the color of it so that it isn't visible anymore?</p>
+    </section>
+</body>
+</html>
```

Comment:
I'm actually super confused about why this is showing up as an addition and not a diff. Did you copy it out of Master branch and paste it into your branch?